crates/maple_cli/src/cmd/tagfiles.rs

```diff
@@ -145,27 +156,25 @@ pub struct TagFiles {
 fn read_tag_files<'a>(
     cwd: &'a PathBuf,
     winwidth: usize,
-    files: &'a Vec<[PathBuf; 2]>,
+    files: &'a [[PathBuf; 2]],
 ) -> Result<impl Iterator<Item = String> + 'a> {
-    let streams = files
+    Ok(files
         .into_iter()
```

Comment:
`into_iter` on a shared reference is the same as `iter`; the latter gives less surprising results and is better here.